### ISSUE TITLE
[FR] Implement ShuffleSplit

### ISSUE THREAD

**Issue by cxhernandez** [msmbuilder/osprey#41]
see: http://scikit-learn.org/stable/modules/generated/sklearn.cross_validation.ShuffleSplit.html#sklearn.cross_validation.ShuffleSplit
